examples/schedulers/manual_event_loop.hpp

```diff
@@ -0,0 +1,163 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ * Copyright (c) NVIDIA
+ *
+ * Licensed under the Apache License Version 2.0 with LLVM Exceptions
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *   https://llvm.org/LICENSE.txt
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#pragma once
+
+#include <stop_token.hpp>
+#include <execution.hpp>
+
+#include <condition_variable>
+#include <mutex>
+#include <type_traits>
+
+namespace example {
+  struct manual_event_loop;
+
+  namespace detail {
+    struct task_base {
+      void execute() noexcept {
+        this->execute_(this);
```

Comment:
`execute_` does not match the spelling `_execute` on line 41

Reply:
It's not referring to that function. It's referring to the `task_base::execute_` data member on line 36.